README.md

```diff
@@ -190,9 +190,22 @@ upholdClient.getTickersByCurrency("BTC").then(new PromiseAction<List<Rate>>() {
 
 ```java
 TransactionDenominationRequest transactionDenominationRequest = new TransactionDenominationRequest("1.0", "BTC");
-TransactionRequest transactionRequest = new TransactionRequest(transactionDenominationRequest, "foo@bar.com");
 
-card.createTransaction(transactionRequest).then(new PromiseAction<Transaction>() {
+// Transaction Destination Request
+TransactionDestinationRequest transactionDestinationRequest = new TransactionDestinationRequest(transactionDenominationRequest, "foo@bar.com");
+
+card.createTransaction(transactionDestinationRequest).then(new PromiseAction<Transaction>() {
+    @Override
+    public void call(Transaction transaction) {
+        // Commit the transaction.
+        transaction.commit();
+    }
+});
+
+// Transaction Origin Request
+TransactionOriginRequest transactionOriginRequest = new TransactionOriginRequest(transactionDenominationRequest, "foo@bar.com");
```

Comment:
Replace `foo@bar.com` by `foobar` since usually the origin is an account id and not an email.